lib/mp4/MP4Parser.ts

```diff
@@ -138,6 +138,18 @@ export class MP4Parser extends BasicParser {
     return Token.readUIntBE(value, 0, value.length);
   }
 
+  /**
+   * Get duration in number of sample of track
+   * @return Number of samples
+   */
+  private static getTrackDuration(track: ITrackDescription): number {
+    let totalDuration = 0;
+    track.timeToSampleTable.forEach(ttst => {
+      totalDuration += ttst.count * ttst.duration / track.soundSampleDescription[0].description.sampleRate;
```

Comment:
This may not exist, and it and I also need to check how to pick which sample-description